modules/react/text/lib/LabelText.tsx

```diff
@@ -17,6 +17,28 @@ const StyledLabel = styled(Text.as('label'))<StyledType & TypeLabelProps>(
   })
 );
 
+/**
+ * ## LabelText
+ * [View Storybook Docs](https://workday.github.io/canvas-kit/?path=/docs/preview-text-react--label-text)
```

Comment:
Can this have a Canvas site link instead? A future URL with a route mapping? @jamesfan may have an idea for where this will go. If we don't plan on having a Canvas site page, I think the Storybook one is fine.

Reply:
All Text Components will have dedicated pages on the Canvas Site once the v8 version of the site is deployed. Label Text will live at `https://canvas.workday.com/components/text/label-text`.

Note that this link will redirect you to the Components Overview page if you visit it behind the company VPN until we resolve the external-to-internal site redirect issue.

Reply:
Yep. Happy to change that. I couldn't remember the new site hierarchy.